### ISSUE TITLE
[CAIA a11y Research - PILOT]: Authenticated Experience Design Patterns Team, Update Prefill Pattern (Nov. 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#95123]
> [!IMPORTANT]
> - **TICKET STATUS:** `Scheduled`
> - **PILOT DATE:** 11/6/2024 at 1:30pm EST
> - **TYPES OF ASSISTIVE TECH:** JAWS
> - **TESTING ENVIRONMENT:** [Codespace](https://bit.ly/researchstudy1124)
> - **RESEARCH MATERIALS:** [Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/authenticated-patterns/Design%20and%20Research/2024-09%20Research%20Initiative%202%20-%20Update%20Prefill/Update%20Prefill%20Research%20Plan.md), [Convo Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/authenticated-patterns/Design%20and%20Research/2024-09%20Research%20Initiative%202%20-%20Update%20Prefill/Update%20Prefill%20Research%20Conversation%20Guide.md) & [Research Goals](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/authenticated-patterns/Design%20and%20Research/2024-09%20Research%20Initiative%202%20-%20Update%20Prefill/Update%20Prefill%20Research%20Plan.md#research-goals)

# CAIA A11y Support
This is a child ticket of the research ticket [94733](https://github.com/department-of-veterans-affairs/va.gov-team/issues/94733) - created to track work, completed from this pilot session.
> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- **Pilot**: @coforma-jamie 
- **Co-pilot**: @sara-amanda 
- **Observer**: @mutiaadhocteam 

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: Authenticated Experience Design Patterns (AEDP)
- OCTO-DE product owner: @beckyphung
- Product manager: @msbtterswrth
- Designer: @ChristineRoseSteiffer + @kristen101606
- FE engineer: @adamwhitlock1 + @bellepx0
- Product/team Slack channel: #tmf-auth-exp-design-patterns

</details>

## Research Goals

<details><summary>Toggle to view team research goals</summary>

- Understand the format that users expect to see prefilled information (“review style” versus gray or white cards)
- Understand if users expect to see their uneditable data in a different format than their editable data
- Understand which language users prefer about where their data is saved and where they expect to see it
- Understand if users want to be able to select where they save their information (form versus profile)
- Understand which language users prefer about where/how to update uneditable information
- Understand if users want to see uneditable data on their review screen or not
- Understand if having data prefilled incentivizes users to login
- Understand if users prefer headers in the prefill alert

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:**  11/6/2024
- **Prototype**: Codespace
- [Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/authenticated-patterns/Design%20and%20Research/2024-09%20Research%20Initiative%202%20-%20Update%20Prefill/Update%20Prefill%20Research%20Plan.md#research-goals) 
- [Research Convo Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/authenticated-patterns/Design%20and%20Research/2024-09%20Research%20Initiative%202%20-%20Update%20Prefill/Update%20Prefill%20Research%20Conversation%20Guide.md) 
</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: @coforma-jamie  will test using JAWS on Chrome using a Windows laptop
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @NaomiPMC  in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11ys**
- [x] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] HOLD placed the week of 10/28/2024 on CAIA Calendar (10/16/2024 placed)
- [x] **Determine date and time for pilot** 11/6/2024
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [x] **Receive Research Goals** add to ticket section above for goals.
- [x] **Review research** [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/authenticated-patterns/Design%20and%20Research/2024-09%20Research%20Initiative%202%20-%20Update%20Prefill/Update%20Prefill%20Research%20Conversation%20Guide.md)
- [x] [Request AT device and tech to be used in sessions] (🔗 `Pending`)
- [x] [Receive AT device and tech from Team in research ticket] (🔗 `Pending`) to be used in sessions to prep for pilot
- [x] **Review prototype** in `Codespace`
- [x] **Run pilot** with VFS Researcher
- [ ] **Optional: Note Taking:** [Notes doc](https://docs.google.com/document/d/16DT_531WYKflsoguz1l-BQHpakrWrgZQtjxBqnrHcv4/edit?usp=sharing)
- [ ] Share findings with team from Pilot doc.
```

**Comment by sara-amanda** [comment]
## Pilot Timing Update
- **Per Jamie's note last week:** The team is on track to submit their research plan early next week (aka this week of 10/28/2024). The plan is to schedule a pilot for the end of the week (of 10/28/2024), and then user sessions the week (or two) after that?
    - [x] Calendar Holds had been set for the Pilot EOW 10/28/2024 (will remove early ones in the week, now)
    - [x] Calendar Holds were set for the Research: Tuesday, November 12⋅9:00am – 5:00pm; Daily, until Nov 16, 2024

@mutiaadhocteam @coforma-jamie

**Comment by coforma-jamie** [comment]
Reviewed the conversation guide and had a couple of notes:
- **Create a shortlink** using bit.ly or whatnot, because...
- **You should ask users how they want to receive the link to the prototype.** Especially for screen reader users, Zoom chat can be finicky to access so they may prefer having the link read aloud to them. It's good to ask, "How do you prefer to receive the link - in Zoom chat, email, or I can read it aloud to you?"
- **"Blue alert" has no meaning to blind participants**, so if any of the AT participants are blind, you'll need another way to ask questions about it.
- **Screen readers don't read bolded text any differently,** so if any participants are blind, the boldness won't mean anything to them. The difference will be in the words, not the formatting.

@kristen101606

**Comment by ChristineRoseSteiffer** [comment]
@coforma-jamie what if, instead of blue alert, we called it 'message at the start of the page' since the alert gets read towards the beginning?

**Comment by coforma-jamie** [comment]
@ChristineRoseSteiffer I think that would work!
